README.md

```diff
@@ -162,14 +162,17 @@ Worklet script (i.e. `reach.js`):
 ```js
 class SendReachReportOperation {
   async function run(data) {
-    // A toy model that only computes reach for users whose favorite color is red.
-    if (data["favorite-color"] != "red") {
+    // A toy model that only computes reach for users who haven't previously had a report sent.
```

Comment:
I'm not sure that the toy model description still applies here. Maybe just something along the lines of "To calculate reach, we skip users who have already had a report sent."?

Alternatively, if we want to keep the toy model description, we could have the if statement be something like `if (await this.sharedStorage.get("report-sent") != "yes") && data["favorite-color"] != "red") {`

Wdyt?

Reply:
I can take out "toy model", no problem.  Will do!